nomad/structs/csi.go

```diff
@@ -862,14 +846,67 @@ func (p *CSIPlugin) Copy() *CSIPlugin {
 	return out
 }
 
+type CSIControllerCapability int
+
+const (
+	CSIControllerSupportsCreateDelete CSIControllerCapability = iota
```

Comment:
Fields parameterized from these values get encoded and saved to disk, right? Similar to how we changed `SnapshotType` to be explicitly numbered, I think we should do the same here to avoid someone accidentally inserting a new value instead of appending.

Reply:
They do, so yeah good call there too. Will fix.

Reply:
Done in e9329a7